src/main/java/de/tum/in/www1/artemis/web/rest/dto/metrics/ExerciseInformationDTO.java

```diff
@@ -0,0 +1,44 @@
+package de.tum.in.www1.artemis.web.rest.dto.metrics;
+
+import java.time.ZonedDateTime;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.Exercise;
+
+/**
+ * DTO for exercise information.
+ *
+ * @param id        the id of the exercise
+ * @param shortName shortTitle the short title of the exercise
+ * @param title     title the title of the exercise
+ * @param start     the start date of the exercise
```

Comment:
Nevermind 🥲 We need a max score after all for the overall performance progress bar. Can you add it? 
![image](https://github.com/ls1intum/Artemis/assets/5898705/6f54af8b-0a49-48c2-b6c0-b9543ca5ab17)